### PR TITLE
Document tools and proposal of a workflow

### PR DESCRIPTION
**Pull request by flip1995**
[Rendered](https://github.com/flip1995/comprakt/blob/workflow/README.md)

### CODE REVIEW

**Comment by reiner-dolp** [comment]
LGTM, I would however add a sentence indicating that `#[rustfmt::skip]` should be used sparingly, e.g. only for tabular constant definitions where you want to align the columns and similar.

File: README.md [hunk 1]
```diff
@@ -11,9 +11,64 @@ A [MiniJava](https://pp.ipd.kit.edu/lehre/WS201819/compprakt/intern/sprachberich
...
+the project via Pull Requests. Every PR should get reviewed by another person to
+ensure keeping the code quality high (typos and simple one-line fixes can be
+accepted by the author). After that the reviewer should use `bors` to merge the
+PR. For big and/or major changes it is recommended to have at least two
+reviewers.
+
+Before creating a PR make sure to run
+
+```bash
+cargo clippy
```
Comment (problame) [inline README.md]: `./build --ci`?

Comment (flip1995) [reply README.md]: Depends on #5. Currently rustfmt is only run with `--check` in the `build` script, which won't format the code.

**Comment by problame** [comment]
bors r=problame,reiner-dolp

**Comment by flip1995** [comment]
https://github.com/comprakt/comprakt/pull/6#issuecomment-430560295 wasn't addressed yet, I'll open a new PR for this.

**Pull request merged**